### DISCUSSION TITLE
What are the plans for Vector?

### DISCUSSION THREAD

**Discussion by omers** [vectordotdev/vector#20782] [General]
Dear Datadog team,

Are there any future plans to close this project and integrate it into Datadog stack?
We have a large deployment stack of Vector, and would be happy to understand what are the commercial future plans of it

Many thanks!

**Comment by jszwedko** [comment]
Hi @omers ,

Datadog doesn't have plans to close this project. There is a newish Datadog product, Observability Pipelines, that builds on top of Vector's offering, but Datadog will continue to support Vector as a community project.

Hope this helps answer your question!

**Reply by omers** [reply]
We appreciate your input, @jszwedko . During our time at an AWS Summit, we picked up some unofficial information from the booth presenters. It's reassuring to learn that Vector's development will continue to be community-driven.
